_posts/2023-08-04-illumsteps.md

```diff
@@ -0,0 +1,363 @@
+
+The easier way to tell if there is uneven illumination in images that are dim or it is just not obvious is to increase the brightness to the point where you can see.
+
+The software that I find to be the best at this (among many other things) is [Fiji](https://imagej.net/software/fiji/.).
+
+It is incredibly simple to use!
+All you have to do is:
+
+1. Open up Fiji
+2. Load in an image from your dataset
+3. Go to "Image -> Adjust -> Brightness/Contrast" (the shortcut on Mac and Linux is `Command + Shift + C`)
+4. Use the second bar to adjust the `maximum` brightness of the image and see what emerges
+
+Once the image is brightened, either you will see the illumination across the image to be even (e.g., as you increase to max brightness, the image will become entirely white) OR you will start to see one part of the image to be more bright than the rest (Figure 2).
+
+![Figure_2](../images/blog/illumsteps/fig_2.png)
+{%
+  include figure.html
+  image="images/blog/illumsteps/fig_2.png"
+  width="100%"
+%}
+
+Now that we have identified that our image definitely needs to be corrected for uneven illumination, we will need to perform an illumination correction function.
+If you would like to go over how to make function in CellProfiler, please go to the [next section of the blog post](https://www.waysciencelab.com/2023/06/21/illumsteps.html#creating-a-cellprofiler-ic-function).
+This section goes over the basics, so lets move on to step two!
+
+### Step 2: Confirm that your IC function worked
+
+Once you have performed the illumination correction method of your choosing, now you should probably confirm that it even worked!
+
+Currently, there is no automatic or exact quantitative way to determine if the IC method worked on your dataset.
+You can probably use ChatGPT and it will tell you to measure the contrast of the image or standard deviation of the pixel intensities in the image, but I find this to be more tedious and does not always work.
+
+The most full-proof method I have found is exactly what I have already told you!
+Brighten up the image and see how the illumination looks across the image.
+What you expect to see is that your IC corrected images will have even illumination across the image and no brighter areas (Figure 3).
+
+![Figure_3](../images/blog/illumsteps/fig_3.png)
+{%
+  include figure.html
+  image="images/blog/illumsteps/fig_3.png"
+  width="100%"
+%}
+
+### Step 3: Repeat for each _channel_
+
+For all datasets you work with, these basic steps should be taken as quality control assurance prior to analysis.
+
+The biggest note is that traditionally, an IC function is created for each `channel`, not all of the images in a dataset. This is due to different distributions and sizes of the objects in a channel.
+As well, I have noticed myself that for some unknown reason to me, there are patterns of uneven illumination to seem to occur only in images in the same channel, but this is just my experience.
```

Comment:
This sentence flows a bit funky, consider rewording. I think it is the double use of myself and me?

Reply:
Now that I am reading it, I am not a fan and don't think it makes sense. Thank you for bringing it up!